README.md

```diff
@@ -43,7 +43,7 @@ Notify a slack channel with a custom message at any point in a job with this cus
 | `include_project_field` | `boolean` | `true` | Whether or not to include the _Project_ field in the message |
 | `include_job_number_field` | `boolean` | `true` | Whether or not to include the _Job Number_ field in the message |
 | `include_visit_job_action` | `boolean` | `true` | Whether or not to include the _Visit Job_ action in the message |
-| `dashboard_link` | `string` |  | Optional dashboard link property for the [Slack message attachment] |
+| `button_links` | `string` |  | Optional dashboard link property for the [Slack message attachment] (space delimited))|
```

Comment:
[]() is a special md syntax for link. You will have to change that. Also you have an extra ')'